.gitignore

```diff
@@ -6,3 +6,4 @@ Gemfile.lock
 .rvmrc
 *.gem
 tags
+.idea
```

Comment:
```suggestion
```

Reply:
Hi @rafaelfranca 
Can I ask you why you don't like this change?

Reply:
Because this file is not part of the project, so it should be configured in your global ignore file, not in the project specific git ignore.

Reply:
Got it 👍